modules/n1ql/pages/n1ql-language-reference/objectfun.adoc

```diff
@@ -19,6 +20,9 @@ SET schedule[0] = {"day" : 7, "special_flights": [
 WHERE destinationairport = "CDG" AND sourceairport = "TLV";
 ----
 
+
+:load-example-warning: pass:q[Make sure that you have run the `UPDATE` command described in <<load-example-data, this section>> before running the example(s).]
```

Comment:
Well, the backticks are now working properly, but the xref shorthand is now replaced by `<>`. See the [preview doc](https://github.com/couchbase/docs-server/blob/70ef31e48ec75e86f1482f5c4fbdf5489126f048/modules/n1ql/pages/n1ql-language-reference/objectfun.adoc).

To work around this, replace the xref shorthand with an xref macro. I've tested this offline and it now seems to work.

 ```suggestion
:load-example-warning: pass:q[Make sure that you have run the `UPDATE` command described in xref:load-example-data[this section] before running the example(s).]
```